src/WpfMath.Tests/WpfMath.Tests.fsproj

```diff
@@ -1,31 +1,30 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
     <PropertyGroup>
-        <TargetFramework>net461</TargetFramework>
+        <TargetFrameworks>netcoreapp3.0;net461</TargetFrameworks>
         <IsPackable>false</IsPackable>
+        <LangVersion>4.7</LangVersion>
```

Comment:
Unfortunately, I have to roll back this particular line due to https://github.com/dotnet/fsharp/issues/7757.

Reply:
Seems that https://github.com/dotnet/fsharp/issues/7757 has been fixed.

Reply:
Yeah, thank you! Not sure how fast it will propagate to any SDK release though. But it has defnintely been fixed :)

Reply:
Still doesn't work in .NET Core 3.1 SDK, so I've rolled it back. We'll use F# 4.7, but won't set the `LangVersion` in the project file, yet.